README.md

```diff
@@ -57,7 +57,7 @@ Large Language Models (LLMs.) The "**lab**" in Instruct**Lab** 🐶 stands for
 
 ## ❓ What is `ilab`
 
-`ilab` is a Command-Line Interface (CLI) tool that allows you to perform the following actions:
+`ilab` is a Core tool that allows you to perform the following actions:
```

Comment:
```suggestion
InstructLab Core is a package that contains the `ilab` command-Line Interface (CLI) tool and allows you to perform the following actions:
```
@nathan-weinberg ^^ this might need an eng tweak

Reply:
^^ I like this as well

Reply:
updated, but `command ` should be `Command`?

Reply:
Yes agreed it should be capitalized